File: Day5(no. of int pairs to a sum val).cpp
#include <iostream>
using namespace std;

int main(){
	int n,x;
	int count=0;
	cout<<"Enter the array size: ";
	cin>>n;
	int arr[n];
	cout<<"Enter the array: ";
	for(int i=0;i<n;i++){
		cin>>arr[i];
	}
	
	cout<<"Enter the sum: ";
	cin>>x;
	
	cout<<"The sum is: ";
	for(int i=0;i<n;i++){
		for(int j=i+1;j<n;j++){
			if(arr[i]+arr[j]==x){
				count++;
			}
		}
	}
	cout<<count;

	return 0;
}

File: Day5(no. of int pairs to a sum val).py
n=int(input("Enter the no. of elements in the array"))
arr=[]
print("Enter the array")
for i in range(n):
    l=int(input())
    arr.append(l)
count=0
x=int(input("Enter the sum val: "))

for i in range(n):
    for j in range(i+1,n):
        if(arr[i]+arr[j]==x):
            count=count+1
print("The no. of val to make the sum is:",count)
